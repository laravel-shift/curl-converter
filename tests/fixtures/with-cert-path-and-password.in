curl -E /path/to/cert:password https://example.com
