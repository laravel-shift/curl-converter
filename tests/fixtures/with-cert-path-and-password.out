Http::withOptions([
        'cert' => [
            '/path/to/cert',
            'password',
        ],
    ])
    ->get('https://example.com');
